### ISSUE TITLE
Would love an option to strip only `devDependencies`

### ISSUE THREAD

**Issue by danoc** [lmammino/package-strip-deps#2]
Hello! I'd like to start stripping only `devDependencies` before publishing packages to NPM.

I can do this by using the `--keep` flag on all of our dependencies, but listing them manually can easily lead to a situation where a new dependency is added but we forget to whitelist it.

Has this feature been considered?

**Comment by lmammino** [comment]
No, but it totally makes sense as a feature. Would you like to attempt to submit a PR for it?

I would think about two flags `--keepDependencies` and `keepDevDependencies` if that makes sense to you

**Comment by danoc** [comment]
Thanks, @lmammino – I actually no longer have a need for this but will come back in the future if that changes. Feel free to close this in the meantime if you'd like.

**Comment by danoc** [comment]
Thanks, @lmammino – I actually no longer have a need for this but will come back in the future if that changes. Feel free to close this in the meantime if you'd like.
